docs/target-compatibility.md

```diff
@@ -56,6 +57,14 @@ tex.GetDimensions is the GetDimensions method on 'texture' objects. This is not
 
 SM6.5 Wave Intrinsics are supported, but requires a downstream DXC compiler that supports SM6.5. As it stands the DXC shipping with windows does not. 
 
+## WaveShuffle
+
+WaveShuffle is an intrinsic added to the Slang stdlibrary to expose the glsl `subgroupShuffle` intrinsics and allow loosened requirements on laneId. 
```

Comment:
I'm not sure I like "stdlibrary" as a term. I might argue for consistently using either "standard library" or "stdlib," and it could probably be argued that the latter should be reserved for less formal documentation (like comments).